params/extras/precompile_config_test.go

```diff
@@ -16,13 +16,17 @@ import (
 	"github.com/ava-labs/subnet-evm/precompile/contracts/rewardmanager"
 	"github.com/ava-labs/subnet-evm/precompile/contracts/txallowlist"
 	"github.com/ava-labs/subnet-evm/utils"
+	"github.com/ava-labs/subnet-evm/utils/utilstest"
 	"github.com/stretchr/testify/require"
 )
 
 func TestVerifyWithChainConfig(t *testing.T) {
 	admins := []common.Address{{1}}
 	copy := *TestChainConfig
 	config := &copy
+	config.AvalancheContext = AvalancheContext{
+		SnowCtx: utilstest.NewTestSnowContext(t),
+	}
```

Comment:
config.SnowCtx = utilstest.NewTestSnowContext(t) 

and similary in all other occurrences